lib/graphql/generators/install_generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'graphql/generators/base_generator'
+module GraphQL
+  module Generators
+    # Add GraphQL to a Rails app with
+    # `rails g graphql:install`.
+    #
+    # Setup a folder structure for GraphQL:
+    #
+    # ```
+    # - app/
+    #   - graphql/
+    #     - resolvers/
+    #     - types/
+    #       - query_type.rb
+    #     - loaders/
+    #     - mutations/
+    #     - {app_name}_schema.rb
```

Comment:
Personally I'm a fan of namespacing the schema name. Rails autoloading expects the filename to match the constant name, and having the app-wide constant for the schema simply be called `Schema` doesn't seem right to me. :/